pkg/reconciler/dependencybuild/dependencybuild.go

```diff
@@ -670,13 +668,21 @@ func (r *ReconcileDependencyBuild) handleBuildPipelineRunReceived(ctx context.Co
 				} else if i.Name == artifactbuild.PipelineResultPassedVerification {
 					parseBool, _ := strconv.ParseBool(i.Value.StringVal)
 					passedVerification = !parseBool
+				} else if i.Name == artifactbuild.PipelineResultVerificationResult {
+					verificationResults = i.Value.StringVal
 				} else if i.Name == artifactbuild.PipelineResultGavs {
 					deployed := strings.Split(i.Value.StringVal, ",")
 					db.Status.DeployedArtifacts = deployed
-				} else if i.Name == artifactbuild.PipelineResultVerificationResult {
-					verificationResults = i.Value.StringVal
+				} else if i.Name == artifactbuild.PipelineResultDeployedResources && len(i.Value.StringVal) > 0 {
+					//we need to create 'DeployedArtifact' resources for the objects that were deployed
+					deployed = strings.Split(i.Value.StringVal, ",")
```

Comment:
Why isn't this using `gavs` from above so that is stored in `run.Results` ?

Reply:
ah, oops, I did not see your question till after I hit the merge button, this is a bug.

Reply:
I'll fix it in #944 :-)

Reply:
Ah, nm, I see you've entered https://github.com/redhat-appstudio/jvm-build-service/pull/945/